packages/babel-traverse/src/path/modification.js

```diff
@@ -126,7 +126,10 @@ export function insertAfter(nodes) {
     if (Array.isArray(this.container)) {
       return this._containerInsertAfter(nodes);
     } else if (this.isStatementOrBlock()) {
-      if (this.node) nodes.unshift(this.node);
+      // Unshift current node if it's not an empty expression
+      if (this.node && (!this.isExpressionStatement() || this.node.expression != null)) {
```

Comment:
Ah ok I see. I guess `this.container['expression'] = null` is rather a weird way to do things but we can do this